services/hanke-service/src/main/resources/application.properties

```diff
@@ -86,6 +86,7 @@ spring.mail.properties.mail.debug=${MAIL_SENDER_DEBUG:false}
 spring.mail.properties.mail.smtp.auth=${MAIL_SENDER_AUTH:false}
 spring.mail.properties.mail.smtp.starttls.enable=${MAIL_SENDER_STARTTLS_ENABLE:true}
 
-haitaton.email.enabled=${HAITATON_EMAIL_ENABLED:false}
+haitaton.email.filter.use=${HAITATON_EMAIL_FILTER_USE:true}
```

Comment:
This should be set to false in docker-compose.yml, so we can still send email to all addresses in the docker environment.

Reply:
Fixed.